jwst/extract_1d/extract.py

```diff
@@ -619,7 +619,7 @@ def update_from_width(
         lower = (lower + upper) / 2. - (width - 1.) / 2.
         upper = lower + (width - 1.)
         ap_width = Aperture(xstart=lower, xstop=upper, ystart=ap_ref.ystart, ystop=ap_ref.ystop)
-
+    
```

Comment:
Looks like extraneous white space has been added on this line. Should be removed.